package.json

```diff
@@ -2,20 +2,21 @@
   "name": "cookie-policy",
   "version": "1.1.0",
```

Comment:
You should probably bump the major version for this change.

Reply:
That'll be a separate PR.